packages/trigger-client/src/dataMappers.ts

```diff
@@ -77,9 +81,21 @@ export function mapServerTriggerCriteria(
       triggerPrice: removeDecimals(criteria.last_price_above),
     };
   }
+  if ('last_price_below' in criteria) {
+    return {
+      type: 'last_price_below',
+      triggerPrice: removeDecimals(criteria.last_price_below),
+    };
+  }
+  if ('mid_price_above' in criteria) {
+    return {
+      type: 'mid_price_above',
+      triggerPrice: removeDecimals(criteria.mid_price_above),
+    };
+  }
   return {
```

Comment:
Should we change this to an explicit `if` check, and error out if there isn't a match?

The caveat though is that future changes will be breaking, but depending how you look at it, that might be preferable